src/envs/blocks.py

```diff
@@ -376,7 +376,8 @@ def _OnTable_holds(state: State, objects: Sequence[Object]) -> bool:
             (desired_z-cls.pick_tol < z < desired_z+cls.pick_tol)
 
     @staticmethod
-    def _GripperOpen_holds(state: State, objects: Sequence[Object]) -> bool:
+    def _GripperOpen_holds(state: State, objects: Sequence[Object]
+                           ) -> bool:
```

Comment:
this is just a new line right? might as well undo, to reduce the number of files involved in the PR